README.md

```diff
@@ -114,6 +114,16 @@ cloudai\
     --tests-dir conf/common/test
 ```
 
+Verify if system configs are valid:
+```bash
+cloudai\
+    --mode verify-systems\
+    --tests-dir s\          # any value here
+    --test-templates-dir d\ # ... and here
```

Comment:
Can we specify directory names based on our current directory structure to maintain consistency with other sections of this README file?

Reply:
Sure, updated.